server/lib/nl/fulfillment/triple.py

```diff
@@ -68,16 +65,23 @@ def populate(uttr: nl_uttr.Utterance) -> bool:
     uttr.counters.err('triple_failed_no_properties', 1)
     return False
 
+  chart_added = False
   for prop in uttr.properties:
     # TODO: handle properties that are links that use ->
     if '->' in prop or '<-' in prop:
       uttr.counters.err('triple_property_unsupported', prop)
       continue
     else:
       if _populate_any_direction_prop(uttr, prop):
+        chart_added = True
         # Currently only handling one prop, so if populate succeeds for any one
         # prop, return.
-        return True
-      else:
-        uttr.counters.err('triple_property_failed_existence', prop)
-  return False
+        break
```

Comment:
Why drop the `else` triple_property_failed_existence counter update?

Reply:
In fact, it might also be worth considering messaging if the top match(es) fail existence? (TODO?)

Reply:
dropped the counter here because I moved to updating the counter now for each direction of the property, so I don't think we need another counter update here? Given a prop A, we will try to get values for both its in and out values where we will also update the triple_property_failed_existence counter if there's no values for each of '->A' and '<-A' .. 

& added todo